projects/table/src/declarations/classes/table-data-display-collection.class.ts

```diff
@@ -17,12 +17,13 @@ interface DistributedColumns {
   rightPinnedColumns: TableColumn[];
 }
 
-const DEFAULT_ROW_HEIGHT_PX: number = 50;
+const DEFAULT_ROW_HEIGHT_PX: number = 40;
 const DEFAULT_MIN_BUFFER_PX: number = 100;
 
 export class TableDataDisplayCollection<T> implements TableDataDisplayCollectionRef<T> {
   public readonly data$: BehaviorSubject<T[]> = new BehaviorSubject<T[]>([]);
-  public readonly selectedIdsList$: BehaviorSubject<string[]> = new BehaviorSubject([]);
+  public readonly selectedRowsIds$: BehaviorSubject<string[]> = new BehaviorSubject([]);
+  public readonly disabledRowsIds$: BehaviorSubject<string[]> = new BehaviorSubject([]);
```

Comment:
Мы полностью указываем типы (где их нет просто давно было написано)
```suggestion
  public readonly disabledRowsIds$: BehaviorSubject<string[]> = new BehaviorSubject<string[]>([]);
```